content/en/docs/reference/using-api/cel.md

```diff
@@ -61,13 +61,38 @@ Example CEL expressions:
+| --------------------------------------------------- | --------------------------------------------------------------------------------------------------------------- | -------------------------- |
+| [standard macros](stdmacros)                        | `has`, `all`, `exists`, `exists_one`, `map`, `filter`                                                           | *                          |
+| [standard functions](stdlib)                        | see the [official list of standard definitions](stdlib)                                                         | *                          |
+| [Homogeneous Aggregate Literals](opt1)              |                                                                                                                 | *                          |
+| [Default UTC Time Zone](opt2)                       |                                                                                                                 | *                          |
+| [Eagerly Validate Declarations](opt3)               |                                                                                                                 | *                          |
+| [extended strings library](stringsgodoc), Version 1 | `charAt`, `indexOf`, `lastIndexOf`, `lowerAscii`, `upperAscii`, `replace`, `split`, `join`, `substring`, `trim` | *                          |
+| Kubernetes list library                             | See below                                                                                                       | *                          |
+| Kubernetes regex library                            | See below                                                                                                       | *                          |
+| Kubernetes URL library                              | See below                                                                                                       | *                          |
+| Kubernetes authorizer library                       | See below                                                                                                       | *                          |
+
+[stdmacros]: https://github.com/google/cel-spec/blob/v0.7.0/doc/langdef.md#macros
+[stdlib]: https://github.com/google/cel-spec/blob/master/doc/langdef.md#list-of-standard-definitions
+[stringsgodoc]: https://pkg.go.dev/github.com/google/cel-go/ext#Strings
+[opt1]: https://pkg.go.dev/github.com/google/cel-go@v0.17.4/cel#HomogeneousAggregateLiterals
+[opt2]: https://pkg.go.dev/github.com/google/cel-go@v0.17.4/cel#DefaultUTCTimeZone
+[opt3]: https://pkg.go.dev/github.com/google/cel-go@v0.17.4/cel#EagerlyValidateDeclarations
+[opt4]: https://pkg.go.dev/github.com/google/cel-go@v0.17.4/cel#OptionalTypes
+[opt5]: https://pkg.go.dev/github.com/google/cel-go@v0.17.4/cel#CrossTypeNumericComparisons
+
+CEL functions, features and language settings support downgrade. For example,
```

Comment:
nit: Is downgrade an action?

Reply:
Yes, I'll switch this to "control plane rollback" to clarify.